website/components/pages/home.tsx

```diff
@@ -51,7 +51,7 @@ export default function Home() {
       <div className="flex flex-col justify-between md:mr-4">
         <div>
           <h1 className="max-w-5xl mx-auto nx-text-6xl font-extrabold tracking-tighter leading-[1.1] sm:text-7xl lg:nx-text-8xl xl:nx-text-8xl">
-             Frustratingly
+             Finally
```

Comment:
```suggestion
             Finally,
```
Shouldn't it have a comma afterwards? @hermanschaaf

Reply:
Yes 👍